### ISSUE TITLE
Load not exactly the same number of weights

### ISSUE THREAD

**Issue by JoeHEZHAO** [shuangshuangguo/caffe2pytorch-tsn#9]
Hello 

Thanks for releasing your work. Big help for someone prefer pytorch like me.

When I check the **test_model.py** code, one thing confused me. I found out that the BNInception has total 485 components. While the ucf101_rgb.pth only recover 416 of them. 

`len(list(net.state_dict()) == 485
  len(list(base_dict)) == 416`

Is this normal ? Does this affect total accuracy ?

Best

**Comment by JoeHEZHAO** [comment]
![bninception_unloaded_weights](https://user-images.githubusercontent.com/14569827/47403584-018ced00-d718-11e8-8df7-f223c2521e01.png)

**Comment by JoeHEZHAO** [comment]
sry I closed it by mistake.

I upload the 69 un-recovered weights name. Seems like they all related to batch track
